tools/cockpit.spec

```diff
@@ -93,6 +93,7 @@ Recommends: %{name}-docker = %{version}-%{release}
 Suggests: %{name}-pcp = %{version}-%{release}
 Suggests: %{name}-kubernetes = %{version}-%{release}
 Suggests: %{name}-selinux = %{version}-%{release}
+Suggests: %{name}-pkgupdates = %{version}-%{release}
```

Comment:
For the scope of this PR, the CLI equivalent is little more than "pkcon refresh" and "pkcon update" (I'll create a page for it). It's going to grow with configuring automatic updates and reboot detection though, as these need backend specific (yum/dnf/apt) commands. But I suppose the main conflict here is "stories/workflow" (apply/configure operating system updates) vs. "backend" (packagekit/dnf/apt/yum).